src/Checkbox.js

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import bootstrapUtils, { bsClass } from './utils/bootstrapUtils';
+
+const propTypes = {
+  inline: React.PropTypes.bool,
+  disabled: React.PropTypes.bool,
+};
+
+const defaultProps = {
+  inline: false,
+  disabled: false,
+};
+
+class Checkbox extends React.Component {
+  render() {
+    const {
+      inline, disabled, className, style, children, ...props,
+    } = this.props;
+
+    delete props.bsClass;
```

Comment:
I don't want to pass it down to the created component, because it's consumed/used by this component for `bootstrapUtils.getClassSet`. I'd pull it out of `this.prop`, but that gives me a variable shadowing error.